package.json

```diff
@@ -38,22 +41,22 @@
     "@types/kafka-node": "^2.0.8",
     "@types/node": "^8.10.44",
     "@types/uuid": "^3.4.3",
-    "@types/winston": "^2.3.9",
+    "@types/winston": "^2.4.4",
     "async": "^2.6.2",
-    "axios": "^0.17.1",
+    "axios": "^0.18.1",
     "body-parser": "^1.18.2",
-    "eslint": "^4.7.2",
+    "eslint": "^4.19.1",
     "express": "^4.16.0",
     "jest": "^23.6.0",
     "jshint": "^2.10.2",
-    "kafka-node": "^2.2.3",
+    "kafka-node": "^2.6.1",
```

Comment:
Does those updates was fully tested?

Reply:
Yes, @muhamedcpqd , they were tested.
The CodeCov check failed, but I am working on tests in another branch.